Frends.MySQL.ExecuteProcedure/Frends.MySQL.ExecuteProcedure/ExecuteProcedure.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Frends.MySQL.ExecuteProcedure.Definitions;
+using MySql.Data.MySqlClient;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Dynamic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Frends.MySQL.ExecuteProcedure;
+
+/// <summary>
+/// Task class.
+/// </summary>
+public class MySQL
+{
+    /// <summary>
+    /// Execute a stored procedure to MySQL.
+    /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.MySQL.ExecuteProcedure)
+    /// </summary>
+    /// <param name="input"></param>
+    /// <param name="options"></param>
+    /// <param name="cancellationToken"/>
+    /// <returns>Object { int AffectedRows }</returns>
+    public static async Task<Result> ExecuteProcedure(
+        [PropertyTab] Input input,
+        [PropertyTab] Options options,
+        CancellationToken cancellationToken
+    )
+    {
+        if(string.IsNullOrWhiteSpace(input.ConnectionString)) throw new ArgumentNullException("Connection string cannot be empty.");
+        if (string.IsNullOrWhiteSpace(input.Query)) throw new ArgumentNullException("Query cannot be empty.");
+
+        try
+        {
+            using (var conn = new MySqlConnection(input.ConnectionString + "UseAffectedRows=true;"))
+            {
+                await conn.OpenAsync(cancellationToken).ConfigureAwait(false);
+
+                IDictionary<string, object> parameterObject = new ExpandoObject();
+                if (input.Parameters != null)
+                {
+                    foreach (var parameter in input.Parameters)
+                    {
+                        cancellationToken.ThrowIfCancellationRequested();
```

Comment:
I don't think there will be so many parameters that cancellation check would be necessary here.